.travis.yml

```diff
@@ -180,7 +180,7 @@ jobs:
         DEP_OPTS="NO_QT=1 NO_UPNP=1 DEBUG=1 ALLOW_HOST_PACKAGES=1"
         RUN_UNIT_TESTS=false
         RUN_FUNCTIONAL_TESTS=true
-        TEST_RUNNER_EXTRA="--tiertwo --sapling --skipcache"   # Run tier two and sapling functional tests only.
+        TEST_RUNNER_EXTRA="--tiertwo --sapling"   # Run tier two and sapling functional tests only.
```

Comment:
Yes good idea (this was done when we didn't have GA yet).
Not strictly "needed", as the few tests using the cache now just recreate it individually, but it will make the job a little faster if we remove that parameter :+1:  
Updated.